### ISSUE TITLE
[Issue] Cant Create OrgAdmin for Default Organisation

### ISSUE THREAD

**Issue by JustCodeIt4Head** [TheHive-Project/Cortex#276]
### Request Type
Issue

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (Debian)       | Debian, Ubuntu, CentOS, RedHat, ...
| OS version (10)       | XP, Seven, 10, Ubuntu, ...
| Cortex version 2.1.3-1   | 2.x, hash of the commit
| Docker              | Docker, Binary, From source
| Browser Chrome 83.0.4103.116    | If applicable

### Problem Description
In previous versions, it was possible to create an Orgadmin for the Standard Cortex Organisation so you could configure the Analyzers, etc. However, in the latest Docker Image Verison (2.1.3-1) it doesn't seem to work anymore. You can only create "Superadmin" Users. After that you can edit the User and change the Role to the Orgadmin, however Updating the User Fails with an Error that the User could not be Updated.
For new Organisations it's Possible to create an Admin, however, you cant see the Analyzers in theHive.

### Steps to Reproduce
1. Login as Default Superadmin User for the Cortex Standard Org
2. Create a New user, select the Default Org
3. Only Available role is Superadmin 

### Possible Solutions
Creating a new Org, however, how do I change the default Org in the default Docker Image installation without Application.conf?

### Complementary information
-

**Comment by JustCodeIt4Head** [comment]
Ok,
so according to the Documentation, the Default Organisation was never intended to be used as an Analyzer Organisation. Dont know if that worked by accident in the previous release or if that got changed.
Anyway, is there any way to change the API Key for the User that gets used for TheHive Instance without having a Custom Application.conf Set in the Docker Instance? or do I have to use a custom File and is there some usable example? The one here: https://github.com/TheHive-Project/TheHive/blob/master/conf/application.sample seems to be outdated sadly

**Comment by garanews** [comment]
Just a question, why you're using cortex 2.1.3 instead 3.0.1?
It is avaiable as docker image:
https://hub.docker.com/r/thehiveproject/cortex/tags
In your docker-compose try to set the version of cortex
`image: 'thehiveproject/cortex:3.0.1'`

**Comment by JustCodeIt4Head** [comment]
> Just a question, why you're using cortex 2.1.3 instead 3.0.1?
> It is avaiable as docker image:
> https://hub.docker.com/r/thehiveproject/cortex/tags
> In your docker-compose try to set the version of cortex
> `image: 'thehiveproject/cortex:3.0.1'`

Well, I used thehiveproject/cortex:latest as my Image since when I checked it for TheHive "latest" was always the latest released Build and i can easily Update my Machines without Changing the Version Numbers around when doing Updates and it was recommended by the Documentation. Was wondering why its Cortex 2.0, but since it had the latest Tag I assumed that the Cortex 3 Integration was either not Supported or in a Beta.

Just checked and apparently the "latest" Tag did not get Updated for over 2 Years. Why the F do they release a new Image for Cortex and don't use the latest Tag to Update, when it's even recommended in the Documentation.

Will Integrate the new Image when I redeploy the Solution with a Native Edge Router, which is planned anyway and I make myself a Reminder that the tagging of the Images is apparently as bad as the Documentation.

So just to get it out of the Way: Can I safely Update to Elastic 6.8.8 or better 7.X or do I still have to use the old 5.6.0 Version as, again, recommended by the Documentation?
